spatialpy/Result.py

```diff
@@ -188,6 +174,7 @@ def get_species(self, species, timepoints=None, concentration=False, determinist
         #t_index_arr = numpy.linspace(0,self.model.num_timesteps,
         #                    num=self.model.num_timesteps+1, dtype=int)
         t_index_arr = self.get_timespan();
+                # Should this ; be here? ^
```

Comment:
Please check line 176-177 before merging.

Reply:
The semicolon definitely should not be there.